storage/remote/queue_manager.go

```diff
@@ -389,6 +389,16 @@ type WriteClient interface {
 	Endpoint() string
 }
 
+type RemoteWriteFormat int64
+
+const (
+	Base1          RemoteWriteFormat = iota // original map based format
+	Min32Optimized                          // two 32bit varint plus marshalling optimization
+	Min64Fixed                              // a single fixed64 bit value, first 32 are offset and 2nd 32 are
+	Min32Fixed
+	MinBytes // two 32bit fixed, similar to optimized but not varints + no manual marshalling optimization
```

Comment:
This is actually two varints


```suggestion
	MinBytes // two varints, similar to optimized but no manual proto marshaling optimization. varints are encoded when building the struct 
```